integration/src/womir_translation.rs

```diff
@@ -667,6 +665,12 @@ impl<'a, F: PrimeField32> Settings<'a> for OpenVMSettings<F> {
                     Directive::Instruction(ib::const_32_imm(output + 1, upper_lo, upper_hi)),
                 ]
             }
+            Op::I32Eqz => {
+                let input = inputs[0].start as usize;
+                let output = output.unwrap().start as usize;
+                return vec![Directive::Instruction(ib::eqi(output, input, 0x0))];
```

Comment:
Fine since you had to write the eq core, but this could be implemented as "x < 1".